embrace-android-sdk/src/integrationTest/kotlin/io/embrace/android/embracesdk/IntegrationTestRule.kt

```diff
@@ -108,7 +110,8 @@ internal class IntegrationTestRule(
                     workerThreadModuleSupplier = { _ -> workerThreadModule },
                     storageModuleSupplier = { _, _, _ -> storageModule },
                     essentialServiceModuleSupplier = { _, _, _, _, _, _, _, _, _ -> essentialServiceModule },
-                    dataCaptureServiceModuleSupplier = { _, _, _, _, _, _, _ -> dataCaptureServiceModule },
+                    dataSourceModuleSupplier = { _, _, _, _, _, _ -> dataSourceModule },
```

Comment:
I tweaked how this works in #716 but I'll rebase off this to make sure your tests still pass